### PR TITLE
EditorUI and TextAlertPopup bindings for mac

### PR DESCRIPTION
**Pull request by SpaghettDev**

### CODE REVIEW

**Comment by SpaghettDev** [comment]
well `SliderThumb::getValue` is inlined on windows but not on mac

File: bindings/2.200/GeometryDash.bro [hunk 1]
```diff
@@ -2884,7 +2884,7 @@ class SliderThumb : cocos2d::CCMenuItemImage {
 	~SliderThumb();
 
 	// inlined on windows
-	float getValue() {
+	float getValue() = mac 0x2e02f0 {
```
Comment (dankmeme01) [inline bindings/2.200/GeometryDash.bro:2887]: sir that is not valid syntax

**Review by dankmeme01 — CHANGES_REQUESTED** [review]

File: bindings/2.200/GeometryDash.bro (hunk 1 above)
Comment (SpaghettDev) [reply bindings/2.200/GeometryDash.bro:2887]: i know, thats why the action is failing. so what is the correct syntax?

Comment (Weebifying) [reply bindings/2.200/GeometryDash.bro:2887]: just `float getValue() = mac 0x2e02f0;`

Comment (SpaghettDev) [reply bindings/2.200/GeometryDash.bro:2887]: its inlined on windows so i'll just effectively be making it unusable on windows, which is a no no

Comment (Weebifying) [reply bindings/2.200/GeometryDash.bro:2887]: dude windows isnt on 2.200

Comment (Cvolton) [reply bindings/2.200/GeometryDash.bro:2887]: just keep the inlined implementation tbh unless youre hooking the func ig

Comment (SpaghettDev) [reply bindings/2.200/GeometryDash.bro:2887]: > dude windows isnt on 2.200
> 

it was

Comment (SpaghettDev) [reply bindings/2.200/GeometryDash.bro:2887]: > just keep the inlined implementation tbh unless youre hooking the func ig

i'll remove the mac binding ig

**Comment by Cvolton** [comment]
gg!

**Pull request merged**